R/bambu-processReads_scoreReadClasses.R

```diff
@@ -2,17 +2,19 @@
 #' @param se summerized experiment object with read classes/ranges
 #' @param genomeSequence genomeSequence
 #' @param annotations GRangesList of annotations
-scoreReadClasses = function(se, genomeSequence, annotations, 
-                                    min.readCount = 2, verbose = FALSE){
+scoreReadClasses = function(se, genomeSequence, annotations, defaultModels, 
+                            fit = TRUE, min.readCount = 2, verbose = FALSE){
     start.ptm <- proc.time()
     options(scipen = 999) #maintain numeric basepair locations not sci.notfi.
+    if (is.character(se)) 
+        se <- readRDS(file = se)
     rowData(se)$GENEID = assignGeneIds(rowRanges(se), annotations)
+    rowData(se)$novel = grepl("gene.", rowData(se)$GENEID)
```

Comment:
shoud we rename this to `novelGene` to avoid any confusion about these being novel transcripts?

Reply:
(not sure if that might be too complicated in terms of downstream changes)

Reply:
done in #6a2ba96ed17eb05fcee50c402e60ab2da35d2d14